librz/core/cmd/cmd_print.c

```diff
@@ -4371,6 +4371,36 @@ RZ_IPI RzCmdStatus rz_print_pattern8_handler(RzCore *core, int argc, const char
 	return RZ_CMD_STATUS_OK;
 }
 
+static void incAlphaBuffer(ut8 *buf, int bufsz) {
+	int i = 0;
+	while (i < bufsz) {
+		buf[i]++;
+		if (buf[i] && isalpha(buf[i])) {
+			break;
+		}
+		if (!buf[i]) {
+			i++;
+			continue;
+		}
+	}
+	// may overflow/hang/end/stop/whatever here
```

Comment:
When running multi-threaded?

Reply:
I'm sure this is old code

Reply:
this is a copy and paste of the functions. i didn't even check what they do.